taggit/managers.py

```diff
@@ -309,13 +339,18 @@ def __lt__(self, other):
         return False
 
     def post_through_setup(self, cls):
-        self.related = RelatedObject(cls, self.model, self)
+        if RelatedObject is not None:  # Django < 1.8
```

Comment:
I'm able to import `RelatedObject` in django 1.8b2 so this logic is no longer working. Could you use django.VERSION for this instead?